src/components/WelcomeTabs/WelcomeTabContent.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import { GrafanaTheme2 } from '@grafana/data';
+import { config } from '@grafana/runtime';
+import { useStyles2 } from '@grafana/ui';
+import { css } from '@emotion/css';
+
+import { CodeSnippet } from 'components/CodeSnippet';
+
+import { SCRIPT_EXAMPLES, TERRAFORM_EXAMPLES } from './constants';
+import { WelcomeTab } from './WelcomeTabs';
+import { dashboardDark, dashboardLight, privateProbeDark, privateProbeLight } from 'img';
+
+interface Props {
+  activeTab: WelcomeTab;
+}
+
+export function WelcomeTabContent({ activeTab }: Props) {
+  const styles = useStyles2(getStyles);
+  switch (activeTab) {
+    case WelcomeTab.Protocol:
+      return (
+        <>
+          <p>Send individual requests via HTTP, DNS, TCP, PING or a traceroute</p>
+
+          <img src={config.theme2.isDark ? dashboardDark : dashboardLight} className={styles.screenshot} />
+        </>
+      );
+    case WelcomeTab.K6:
+      return (
+        <>
+          <p>Use k6 scripts to monitor your services flexibly</p>
+          <CodeSnippet
+            canCopy={true}
+            className={styles.codeSnippet}
+            tabs={[
+              {
+                value: 'Example scripts',
+                label: 'k6 Examples',
+                groups: SCRIPT_EXAMPLES.map(({ label, script }) => ({
+                  value: label,
+                  label,
+                  code: script,
+                  lang: 'js',
+                })),
+              },
+            ]}
+            dedent={true}
+            lang="js"
+            initialTab="initialize"
+            code="console.log('hello world')"
+          />
+        </>
+      );
+    case WelcomeTab.PrivateProbes:
+      return (
+        <>
+          <p>
+            In addition to the locations we provide out of the box, you can set up your own probes to run inside your
+            network or from a location of your choosing
+          </p>
+          <img src={config.theme2.isDark ? privateProbeDark : privateProbeLight} className={styles.screenshot} />
```

Comment:
🚨 Forgot to leave my other accessibility police comment: Alt text (for all the images), please 😄